### PR TITLE
Prevent speed-bump to insert two ads at the same place.

### PR DESCRIPTION
**Pull request by noppanit**
#19

### CODE REVIEW

File: inc/class-speed-bumps.php [hunk 1]
```diff
@@ -60,7 +60,8 @@ public static function insert_speed_bumps( $the_content ) {
 						'index' => $index,
 						'speed_bump_id' => $id,
 						'inserted_content' => $content_to_be_inserted,
-						);
+					);
+					unset( Speed_Bumps::$_speed_bumps_args[ $id ] );
```
Comment (noppanit) [inline inc/class-speed-bumps.php]: Not sure if this is allowed in PHP world. But I'm trying to delete the inserted element so it won't get checked again.

**Comment by noppanit** [comment]
@goldenapples #reviewmerge once CI passed please

File: inc/class-speed-bumps.php (hunk 1 above)
Comment (goldenapples) [inline inc/class-speed-bumps.php]: Why can't you just call `continue` here? Modifying an array as you're looping through it seems very messy. Also, what about the case of a speed bump that can be conceivably inserted more than once.

Comment (noppanit) [inline inc/class-speed-bumps.php]: You're right I'll change the code.

**Comment by noppanit** [comment]
@goldenapples hmm I'm still seeing two speed bumps inserted on my local machine. Going to do some more debugging.

**Comment by noppanit** [comment]
@goldenapples actually no it's working. I just need to change the code in theme. If this PR is ok please merge.

**Comment by danielbachhuber** [comment]
@noppanit Can you address #22?

@goldenapples If you want to merge these plugin pull requests, can you create corresponding theme PRs for the final build? It's hard for me to keep track of what needs to go to VIP when I don't do the final merge.

**Comment by goldenapples** [comment]
> can you create corresponding theme PRs for the final build.

Yes, I will. As some of the recent changes have introduced breaking changes, I've been stepping back so @noppanit can keep the theme PR's updated along with plugin changes as necessary. See https://github.com/fusioneng/fusion-theme/pull/3201 I think the plugin is at a stable enough point now that I can start updating the submodule in our theme atomically.

**Comment by noppanit** [comment]
@goldenapples Should this still be merged?

**Pull request merged**